src/static/css/nav-primary.less

```diff
@@ -0,0 +1,487 @@
+        }
+
+        & .list_item a {
+            display: block;
+            padding: (@grid_gutter-width / 2)
+                     @grid_gutter-width
+                     (@grid_gutter-width / 2)
+                     (@grid_gutter-width / 2);
+            position: relative;
+            background: @gray-20;
+            font-size: unit(18px / @base-font-size-px, em);
+        }
+        & > .list_item > a:hover {
+            background: @white;
+            color: @darkgray;
+        }
+        & > .list_item > a:focus {
+            background: @white;
+            color: @darkgray;
+        }
+        & > .list_item > a:active {
+            background: @gray-10;
+            color: @darkgray;
+        }
+
+        & .list_item .list_link__disabled {
+            background: @gray-20 !important;
+        }
+
+        // Two levels deep.
+        & .list-expanding .list-expanding_list-item a {
+            padding-left: @grid_gutter-width;
+            background: @gray-10;
+            font-size: 1em;
+        }
+
+        // Three levels deep.
+        & .list-expanding .list-expanding .list-expanding_list-item a {
+            padding-left: @grid_gutter-width * 1.5;
+            background: @gray-5;
+        }
+
+        // Give the expanding trigger an icon.
+        .list-expanding_trigger:after {
+            position: absolute;
+            right: @grid_gutter-width / 2;
+            font-family: 'CFPB Minicons';
+            font-style: normal;
+            font-weight: normal;
+            -webkit-font-smoothing: antialiased;
+        }
+
+        .list-expanding_trigger[aria-pressed="false"]:after {
+            content: "\e006";
+        }
+
+        .list-expanding_trigger[aria-pressed="true"]:after {
+            content: "\e004";
+        }
+
+        // Hide a child list if the trigger is not pressed.
+        .list-expanding_trigger[aria-pressed="false"] .list-expanding_child-list {
+            display: none;
+        }
+    }
+
+    .sliding-nav {
+        .u-clearfix();
+        // Putting the navigation background here instead of on .sliding-nav_nav
+        // is a trick to get the bg of the nav to match the height of .sliding-nav_page.
+        background: @gray;
+    }
+    .sliding-nav.is-open {
+        overflow: hidden;
+    }
+
+    .sliding-nav_page {
+        box-sizing: border-box;
+        width: 100%;
+        float: left;
+        background: @white;
+        transition: .2s margin-right ease-in, .2s box-shadow ease-in;
+    }
+    .is-open .sliding-nav_page {
+        border-left: 1px solid @gray-80;
+        margin-right: -80%;
+        position: relative;
+        box-shadow: -6px 0 0 rgba(16, 24, 32, 0.2);
+    }
+
+    // This overlays the main content and when clicked closes the menu.
+    .sliding-nav_page-overlay {
+        position: relative;
+    }
+    .sliding-nav_page-overlay:before {
+        position: absolute;
+        z-index: 999;
+        // background: @black;
+        opacity: 0;
```

Comment:
So this goes back to https://github.com/cfpb/cfgov-refresh/pull/151. Looks like the markup was never added. I'm going to comment out the whole block for now.